packages/sanes-chrome-extension/src/routes/recovery-phrase/test/travelToRecoveryPhrase.ts

```diff
@@ -0,0 +1,25 @@
+import TestUtils from 'react-dom/test-utils';
+import { Store } from 'redux';
+import { RECOVERY_PHRASE_ROUTE, WELCOME_ROUTE } from '../../paths';
+import { createDom } from '../../../utils/test/dom';
+import { history } from '../../../store/reducers';
+import { whenOnNavigatedToRoute } from '../../../utils/test/navigation';
+import { sleep } from '../../../utils/timer';
+
+export const travelToRecoveryPhrase = async (
+  store: Store
+): Promise<React.Component> => {
+  const dom = createDom(store);
+  TestUtils.act(
+    (): void => {
+      history.push(WELCOME_ROUTE); //I need this to test "Back" button behaviour
```

Comment:
What happens if you remove it?

Reply:
Then history stack will be empty when `history.goBack();` will be called. And it will be not possible to check for route (or maybe it will point to `/`)